fuzz/corpora/conf/0d7ad6e04c0235cdc590756ceec867a05cff5823

```diff
@@ -0,0 +1,41 @@
+=;2I8
+=$$$$$$󠁉
+=$$$$$$$
+=$$$
+=$$$󠁷
+=$$$
+=$$$
+=$$$
+=
+=$$$
+=$$$
+=$$$󠁷
+=$$$
+=$$$
+=$$$
+=$$$
+=$$$$$$$
+=$$$
+=$$$
+=$$$
+=$$$
+=$$$
+=$$$
+=$$$$$$$
+=$$$
+=$$$
+=$$$
+=$$$
+=$$$
+=$
+=$$$
+=$$$$$$$
+=$$$
+=$󠁝$$
+=$$$
+=$$$
+=$$$
+=$$$
+=$$$
+=$$$
+=$$$$$
```

Comment:
There are a few strange quirks in this file (there are some invisible non-printed characters in there as well). Its just the auto-generated fuzz file created by oss-fuzz. I just inserted it as-is without any amendments.